packages/cli/src/rpc.ts

```diff
@@ -121,6 +105,32 @@ export class RPCManager {
       this.log(`TEST PONG received from ${encodedENR.nodeId}`)
       return `PING/PONG successful with ${encodedENR.nodeId}`
     },
+    portal_history_findContent: async (params: [string, Uint8Array]) => {
+      const [enr, contentKey] = params
+      await this._client.sendFindContent(enr, contentKey, SubNetworkIds.HistoryNetwork)
+    },
+    portal_history_offer: async (params: [string, string[], number[]]) => {
+      const [dstId, blockHashes, contentTypes] = params
+      contentTypes.forEach((contentType) => {
+        try {
+          isValidId(dstId)
+          contentType > 0
+          contentType < 2
+        } catch {
+          throw new Error('invalid parameters')
+        }
+      })
+      const contentKeys = blockHashes.map((blockHash, idx) => {
+        return HistoryNetworkContentKeyUnionType.serialize({
+          selector: contentTypes[idx],
+          value: {
+            chainId: 1,
+            blockHash: fromHex(blockHash.slice(2)),
+          },
+        })
+      })
+      await this._client.sendOffer(dstId, contentKeys, SubNetworkIds.HistoryNetwork)
```

Comment:
Here again, let's return something from this RPC call (presumably the contents of the ACCEPT response or else an empty array if we don't get anything)